src/mono/netcore/System.Private.CoreLib/src/System/RuntimeType.Mono.cs

```diff
@@ -1937,19 +1935,19 @@ internal override FieldInfo GetField(FieldInfo fromNoninstanciated)
             RuntimeConstructorInfo? ctor = GetDefaultConstructor();
             if (!nonPublic && ctor != null && !ctor.IsPublic)
             {
-                throw new MissingMethodException(SR.Format(SR.Arg_NoDefCTor, FullName));
+                throw new MissingMethodException(SR.Format(SR.Arg_NoDefCTor, this));
```

Comment:
there's another usage of `SR.Arg_NoDefCTor` that still uses `FullName` in `CreateInstanceForAnotherGenericParameter()`, should that one be switched over to `this` as well?

Reply:
Updated as well